public/static/docs/command-reference/update.md

```diff
@@ -27,18 +27,16 @@ Note that import stages are considered always locked, meaning that if you run
 `dvc repro`, they won't be updated. `dvc update` is the only command that can
 update them.
 
-Another detail to note is that when the `--rev` (revision) option of
-`dvc import` has been used to create an import stage, DVC is not aware of what
-kind of
-[Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References) this
-is, for example a branch or a tag. For typically static references (e.g. tags),
-or for SHA commits, `dvc update` will not have any effect on the import. Refer
-to the
-[re-importing example](/doc/command-reference/import#example-fixed-revisions-re-importing)
-to learn how to "update" fixed-revision imports.
+To "update" fixed-revision to static references (e.g. tags), or for SHA commits,
+use `dvc update` with the `--rev` option.
 
 ## Options
 
+- `--rev` - specific
+  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+  (such as a branch name, a tag, or a commit hash) of the DVC repository to
+  update the data from. Using this option also updates the fixed-revision.
```

Comment:
> Using this option also updates the fixed-revision.

OK but maybe we should explain what a fixed revision is and what exactly gets updated in the DVC-file (import stage), instead of using that term. Please lmk if you'd like some suggestions, I can def. help with the writing part if needed.

Reply:
For inspiration check out this not, found in the **Examples** section below:

> Note that `dvc update` updates the `rev_lock` field of the corresponding [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring in).